examples/02_classification.ipynb

```diff
@@ -129,12 +137,12 @@
     "* 1096 individual time series instances = single days of total power demand (mean subtracted)\n",
     "* one single variable per time series instances, `total_power_demand`\n",
     "    * total power demand on that day, in that hourly period\n",
-    "    * hence a univariate dataset\n",
+    "    * Since there's only one column - `total_power_demand `it is a univariate dataset\n",
```

Comment:
agreed but maybe can be shortened: "there is only one column - etc"

Reply:
Changed as requested to "Since there's only one column - it is a univariate dataset"